docs/04-guides/playback-a-video-stream.mdx

```diff
@@ -1,202 +1,21 @@
-
-> **Note:** All sessions for a stream have the same playback URL.
-
-## Ways to Get the Playback URL
-
-- Via the API (`cURL` or `JavaScript`)
-- Via the [Dashboard](https://livepeer.studio/dashboard)
-
-## Get Playback URL with the GET request to the API
-
-Make a `GET` request to `https://livepeer.studio/api/stream/{id}`.
-
-- Note the value of `playbackId` in the response.
-- The playback URL is `https://livepeercdn.com/hls/{playbackId}/index.m3u8`.
-
-## cURL Method
-
-### Request
-
-```bash
-curl --location --request GET 'https://livepeer.studio/api/stream/{id}' \
---header 'Authorization: Bearer {api_key}'
-```
-
-### Response
-
-```json
-{
-  "lastSeen": 1654285662946,
-  "isActive": false,
-  "record": true,
-  "suspended": false,
-  "sourceSegments": 2041,
-  "transcodedSegments": 8168,
-  "sourceSegmentsDuration": 4074.216000000001,
-  "transcodedSegmentsDuration": 16304.863999999987,
-  "sourceBytes": 1886568720,
-  "transcodedBytes": 1175130472,
-  "id": "405a4ab4-9c5f-48a0-859a-4bb6445994a7",
-  "kind": "stream",
-  "name": "Test Stream",
-  "region": "mdw",
-  "userId": "eada599f-3d58-499b-ba24-c7f3faf988de",
-  "profiles": [
-    {
-      "fps": 0,
-      "name": "240p0",
-      "width": 426,
-      "height": 240,
-      "bitrate": 250000
-    },
-    {
-      "fps": 0,
-      "name": "360p0",
-      "width": 640,
-      "height": 360,
-      "bitrate": 800000
-    },
-    {
-      "fps": 0,
-      "name": "480p0",
-      "width": 854,
-      "height": 480,
-      "bitrate": 1600000
-    },
-    {
-      "fps": 0,
-      "name": "720p0",
-      "width": 1280,
-      "height": 720,
-      "bitrate": 3000000
-    }
-  ],
-  "createdAt": 1651776454667,
-  "streamKey": "405a-lacy-ynpa-dnq2",
-  "ingestRate": 0,
-  "playbackId": "405ag6i16yojk2un",
-  "renditions": {},
-  "multistream": {
-    "targets": []
-  },
-  "outgoingRate": 0
-}
-```
-
-## JavaScript Method
-
-### Request
-
-```javascript
-const axios = require("axios");
-
-axios
-  .get("https://livepeer.studio/api/stream/{id}", {
-    headers: {
-      Authorization: "Bearer {api_key}",
-    },
-  })
-  .then((res) => {
-    console.log(JSON.stringify(res.data));
-  })
-  .catch((error) => {
-    console.log(error);
-  });
-```
-
-### Response
-
-Same as the cURL's method
-
-## Get the Playback URL Through the Dashboard
-
-1. Log in to Livepeer Studio Dashboard
-1. Navigate to the [Streams page](https://livepeer.studio/dashboard/streams)
-1. Click on a stream name
-1. The playback URL is listed in the details section of the page.
-
-## Video player
-
-To build an application that plays your livestream or On Demand assets, you need
-to use some sort of video player compatible with the HLS protocol. There are
-many HLS video players out there, for example [video.js](https://videojs.com/)
-or [JW Player](https://www.jwplayer.com/) which either have built-in support or
-have plugins to support HLS.
-
-### Embed a video player
```

Comment:
Any reason to remove the section about embedding the video player? IIUC it was a requirement in the original issue to have it. We could probably write a simplified version of it (especially now that we have the embed button in the dashboard), but I think we should still mention it here. wdyt? cc @adamsoffer @pglowacky